### DISCUSSION TITLE
大佬，宝塔部署完了  咋老是加载动画中   进不去网页

### DISCUSSION THREAD

**Discussion by yanxscn** [imsyy/home#98] [Q&A]
![image](https://user-images.githubusercontent.com/126781690/222707183-8956b242-3ec5-4c7f-942e-39eb64c1110e.png)

**Comment by 176822** [comment]
> ![image](https://user-images.githubusercontent.com/126781690/222707183-8956b242-3ec5-4c7f-942e-39eb64c1110e.png)
> 
> 

我也是来到这一步了

**Reply by 176822** [reply]
求作者莅临指导🙏

**Reply by imsyy** [reply]
我稍后写个教程吧

**Reply by imsyy** [reply]
> 求作者莅临指导🙏

https://github.com/imsyy/home/discussions/104

**Reply by yanxscn** [reply]
我已成功搭建，感谢作者
